src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,22 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movie")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @OneToMany(fetch = FetchType.EAGER)
```

Comment:
OneToMany relationship with Actor is not fully defined. You should specify the mappedBy attribute to tell Hibernate which field on the other side of the relationship is responsible for this relationship.